admin/cmd/bootstrap.go

```diff
@@ -187,6 +190,10 @@ func finishBootstrap(globalFlags *flags.GlobalFlags) {
 
 	agentlocal.SetTransport(ctx, globalFlags.EnableDebug || globalFlags.EnableTrace, globalFlags.PMMAgentListenPort)
 
+	if !globalFlags.DisableVersionCheck {
+		checkVersionCompatibility(bool(globalFlags.JSON))
```

Comment:
Would it make sense to run this in a go routine not to block the bootstrap?

Reply:
I think even with a goroutine, we'll still have to wait for it to return before proceeding to run the invoked command, so I'm not sure it offers an advantage here.

Reply:
I think we don't have to wait for the output. The only outcome of the check seems to be a message in the log which can come at any point without slowing down the boot process.
Is there maybe anything I missed?